WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -701,20 +791,80 @@ public void onFailure(int statusCode) {
     }
 
     /*
+     * post slugs resolution to IDs has completed
+     */
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ReaderEvents.PostSlugsRequestCompleted event) {
```

Comment:
As a precaution, we should make sure this event is for the post currently being displayed.